scipy/stats/_stats_mstats_common.py

```diff
@@ -60,6 +60,7 @@ def linregress(x, y=None):
     --------
     >>> import matplotlib.pyplot as plt
     >>> from scipy import stats
+    >>> import numpy as np
```

Comment:
In all SciPy examples, this line is implicitly assumed to avoid rewriting it hundreds of times hence we always omit it.

Reply:
I see. Cheers. I did wonder, and I thought it best to include it so it doesn't fail doctests.